### PR TITLE
Assay name

### PR DESCRIPTION
**Pull request by antagomir**
To be merged only after the scater changes in https://github.com/Alanocallaghan/scater/pull/194 have been merged.

### CODE REVIEW

File: .github/workflows/check-bioc.yml [hunk 1]
```diff
@@ -1,270 +0,0 @@
...
-## https://lcolladotor.github.io/biocthis/articles/biocthis_dev_notes.html
-##
-## You can add this workflow to other packages using:
-## > biocthis::use_bioc_github_action()
-##
-## Using GitHub Actions exposes you to many details about how R packages are
-## compiled and installed in several operating system.s
-### If you need help, please follow the steps listed at
-## https://github.com/r-lib/actions#where-to-find-help
-##
```
Comment (TuomasBorman) [inline .github/workflows/check-bioc.yml:14]: Was this intentional?

File: DESCRIPTION [hunk 2]
```diff
@@ -1,13 +1,13 @@
 Package: miaViz
 Title: Microbiome Analysis Plotting and Visualization
-Version: 1.7.1
+Version: 1.7.3
```
Comment (TuomasBorman) [inline DESCRIPTION:3]: 1.7.2

File: vignettes/miaViz.Rmd [hunk 3]
```diff
@@ -328,7 +329,7 @@ plotDMNFit(dmn_se, type = "laplace")
 
 # Serial data ordination and trajectories
 
-```{r, error=FALSE, warning=FALSE, results='hide'}
+```{r, error=FALSE, warning=FALSE, results='hide', eval=FALSE}
```
Comment (TuomasBorman) [inline vignettes/miaViz.Rmd:332]: Build currently fails "miaTime not found" I guess this is good to have for now (we can also have it behind require)

**Review by TuomasBorman — COMMENTED** [review]
Also mia logo got removed

File: .github/workflows/check-bioc.yml (hunk 1 above)
Comment (antagomir) [reply .github/workflows/check-bioc.yml:14]: Not really. I should have paid more attention here.

**Pull request merged**